pkgs/by-name/tr/traccar/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "traccar";
+  version = "6.5";
+
+  src = fetchzip {
+    stripRoot = false;
+    url = "https://github.com/traccar/traccar/releases/download/v${version}/traccar-other-${version}.zip";
+    hash = "sha256-XCG3G24oe/qR6LiMJASb9STOnyTCtw+2HigaPawcQvU=";
+  };
+
+  installPhase = ''
```

Comment:
should include preInstall and postInstall

Reply:
:+1: